examples/tutorials/05_creating_CNN.ipynb

```diff
@@ -13,8 +13,7 @@
    "source": [
```

Comment:
<p>Use shorter names: model, channels</p><p>image_shape is not needed at all</p><p> </p><p>Moreover, you don't need to choose framework either. Just state the model used.</p><p>Write a detailed description about models and frameworks, provide a short inheritance example</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/476/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>